src/types/shell/xdg_shell_v6.rs

```diff
@@ -84,7 +85,7 @@ impl XdgV6ShellSurface {
                                               } });
         (*shell_surface).data = Box::into_raw(shell_state) as *mut _;
         XdgV6ShellSurface { liveliness: Some(liveliness),
-                            state,
+                            state: state,
```

Comment:
No need to write out `state: state` explicitly, use the feature I use everywhere else where you can just use the variable name.

Reply:
Yeah, I had to use state.into(), but as state is cloned before that, i opted to replace state with state.into() at the start of the function